.gitignore

```diff
@@ -104,6 +104,8 @@ env2/
 
 # vim swp
 *.swp
+# emacs bkup
+*~
```

Comment:
Personally I'd add such ignore patterns to my global settings and restrict the .gitignore of the project to patterns which are specific for the project. But I know that there are different opinions.

Reply:
Sure, that would be better practice, but if the project-specific `.gitignore` prevents any new contributor from accidentally comitting unwanted files, I think it's fine to add at least vim/emacs in addition to the other python-specific patterns.